include/kernel.h

```diff
@@ -1729,13 +1729,14 @@ __deprecated static inline void k_disable_sys_clock_always_on(void)
 /**
  * @brief Get system uptime (32-bit version).
  *
- * This routine returns the lower 32-bits of the elapsed time since the system
- * booted, in milliseconds.
+ * This routine returns a value obtained by converting the lower
+ * 32-bits of the system clock into milliseconds.
```

Comment:
This change looks like it got left in by mistake.  In fact the effect of this patch is to make the old text true (where it wasn't before).  The new text reflects the old behavior.

Reply:
Oh wait, no, the old text wasn't right either.  Suggest "This routine returns the lower 32 bits of the system uptime in milliseconts".  It's not the result of converting only the lower 32 bits anymore.

Reply:
Replaced with suggested text.